src/LibVLCSharp.WPF/User32Wrapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
these probably belong to `MarshalUtils.cs`

Reply:
Agreed

Reply:
MarshalUtils is internal static class, I don't think it's good idea to make it public.

Reply:
We use internalsToVisible for others, not sure it's worth it here though, let me think about it.